src/components/MenuBar.js

```diff
@@ -17,7 +17,7 @@ const styles = theme => ({
   },
 });
 
-class SimpleMenu extends Component {
+class HeaderUserMenu extends Component {
```

Comment:
Rename 'HeaderUserMenu' to 'Menu'. Your file name and class name should be consistency.

Reply:
changed to UserMenu